website/content/docs/release-notes/1.17.0.mdx

```diff
@@ -13,12 +13,13 @@ description: |-
 
 ## Important changes
 
-| Change                  | Description
-|-------------------------|------------
-| New default             | [Allowed audit headers now have unremovable defaults](/vault/docs/upgrading/upgrade-to-1.17.x#audit-headers)
-| Opt out feature         | [PKI sign-intermediate now truncates `notAfter` field to signing issuer](/vault/docs/upgrading/upgrade-to-1.17.x#pki-truncate)
-| Beta feature deprecated | [Request limiter deprecated](/vault/docs/upgrading/upgrade-to-1.17.x#request-limiter)
-| Known issue             | [PKI OCSP GET requests can return HTTP redirect responses](/vault/docs/upgrading/upgrade-to-1.17.x#pki-ocsp)
+| Version | Change                  | Description                                                                                                                    |
```

Comment:
I'm happy to remove or adjust this column if necessary, it just felt like the right way to indicate that this was 1.17.0 only and will be fixed in the next version. Let me know what you think or if there's another approach you'd prefer I take for such an issue.

<img width="718" alt="image" src="https://github.com/hashicorp/vault/assets/1594272/6df95eb4-a279-4fc0-b0d4-aca44086d76d">


It can be viewed here: https://vault-54m6izb8b-hashicorp.vercel.app/vault/docs/release-notes/1.17.0